src/classes/CRLP_RollupAccount_TEST.cls

```diff
@@ -249,8 +256,21 @@ private class CRLP_RollupAccount_TEST {
 
         insert opps;
 
+        // create 2 $10 Account Soft Credits
+        Integer scAmount = 10;
+        Integer ascCount = 2;
+        List<Account_Soft_Credit__c> accSCs = new List<Account_Soft_Credit__c>();
+        for (Integer i=0; i<ascCount; i++) {
+            Account_Soft_Credit__c accSC = new Account_Soft_Credit__c (
```

Comment:
Is it worth setting a Role value and validating the filter rule against the Role?